qt/python/mantidqt/widgets/algorithmselector/widget.py

```diff
@@ -57,12 +58,13 @@ def _make_execute_button(self):
 
     def _make_search_box(self):
         """
-        Make a algorithm search box.
+        Make an algorithm search box.
         :return: A QComboBox
         """
         search_box = QComboBox(self)
         search_box.setEditable(True)
         search_box.completer().setCompletionMode(QCompleter.PopupCompletion)
+        search_box.completer().setFilterMode(Qt.MatchContains)
```

Comment:
The `AlgorithmSelector` is in the `mantidqt` library and is currently compiled for both Qt4 and Qt5 so we can use it places transition of interfaces with MantidPlot. We'll need to protect this line with a check for Qt >= 5.2. It's best to use `LooseVersion` e.g. [requirements.py](https://github.com/mantidproject/mantid/blob/14e0b436a905f50475bf6e8e4c5ab248dc9cc8e4/qt/applications/workbench/workbench/requirements.py#L29) for this.

Reply:
Would it be better to use the `qtpy.PYQT5` flag (which is set to `True` when Qt5 is used), instead of checking `qtpy.PYQT_VERSION`?

Reply:
Absolutely. Thanks for pointing at that it exists!